README.md

```diff
@@ -40,15 +40,22 @@ If you would like to run the frontend in a Docker image (e.g. to isolate depende
 ## Unit Testing
 
 Unit tests are co-located with source code with naming convention `*.unit.test.js`.
-Jest is the test driver. Unit tests are isolated from integration dependencies via mocks; including Web3 RPC APIs and smart contract interactions.
-No local blockchain node is expected to run for unit testing. Hard Hat is not required.
 
-To run all unit test and see coverage report:
+We use the [React Jest](https://jestjs.io/docs/tutorial-react) test driver.
+
+
+To run all unit test and see a coverage report:
 
 ```
 yarn test:unit
 ```
 
+Note that the focus of unit testing is to excersize all paths through the code hosted in this repo and **only** code hosted in this repo. To the extend possible, unit tests should abstract out dependencies such as remote API calls as well as crypto wallet APIs via [`mock functions`](https://jestjs.io/docs/mock-functions).
```

Comment:
typos: exercise, extent

Reply:
Hey, I thought we are using decentralized community English.